roles/cockpit_run_verify/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# vim: set ft=ansible:
+#
+- name: Set cockpit container name for RHELAH
+  set_fact:
+    cockpit_cname: "registry.access.redhat.com/rhel7/cockpit-ws"
+  when: ansible_distribution == "RedHat"
+
+- name: Set cockpit container name for Fedora/CentOS
+  set_fact:
+    cockpit_cname: "cockpit/ws"
+  when: ('CentOS' in ansible_distribution) or ansible_distribution == "Fedora"
+
+- name: Run cockpit container
+  command: atomic run {{ cockpit_cname }}
```

Comment:
@cgwalters The timeout only applies to the waiting for port 9090 to open.  In general, we have not explicitly set timeouts for commands to run in Ansible.  This is just a simple test to verify the cockpit container can be run.  

Is there a reason why you want to see a separate docker pull and atomic run?  There will be specific docker and atomic tests added to the improved sanity tests in the near future so we will have coverage for the individual commands.  @miabbott is working on the docker sanity tests right now.